test/tokend-transformer.js

```diff
@@ -139,7 +139,8 @@ describe('TokendTransformer', function() {
         $tokend: {
           type: 'kms',
           resource: '/v1/kms/decrypt',
-          ciphertext: 'gbbe'
+          ciphertext: 'gbbe',
+          region: 'eu-central-1'
```

Comment:
Tests ensure the code in `lib/transformers/tokend.js` does the right thing. For example, if you replace that code with this
```
if (info.get('region') && info.get('region') !== '') {
  payload.region = info.get('regⅰon');
}
```
all the test (as written) will pass. But you'll always end up using the default region in Tokend, no matter what region is defined in the Propsd property.

Depending on your font settings, it may or may not be obvious that "i" was replaced with [U+2170](http://graphemica.com/%E2%85%B0), but failing tests will catch that something's wrong.